src/net_processing.cpp

```diff
@@ -218,7 +241,10 @@ struct Peer {
     /** Work queue of items requested by this peer **/
     std::deque<CInv> m_getdata_requests GUARDED_BY(m_getdata_requests_mutex);
 
-    explicit Peer(NodeId id) : m_id(id) {}
+    explicit Peer(NodeId id, bool addr_relay)
+        : m_id(id)
+        , m_addr_known{addr_relay ? nullptr : std::make_unique<CRollingBloomFilter>(5000, 0.001)}
+    {}
```

Comment:
Out of curiosity, is there a way to teach the [`clang-format-diff.py`](https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy) to support such formatting, i.e., both a leading comma and `{}` on the the same line?

Reply:
Good question! I'm not an expert on clang-format.

Reply:
Perhaps `BCIS_BeforeComma`? (https://clang.llvm.org/docs/ClangFormatStyleOptions.html)